platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultConfigurationCache.kt

```diff
@@ -327,7 +327,7 @@ class DefaultConfigurationCache internal constructor(
                     val description = formatBootstrapSummary(
                         "%s as configuration cache cannot be reused because %s.",
                         buildActionModelRequirements.actionDisplayName.capitalizedDisplayName,
-                        checkedFingerprint.reason
+                        checkedFingerprint.reason.asString()
```

Comment:
Not sure how to explain that, from the stacktrace alone I couldn't figure out, but here or at the call to `String.format(...)`, `toString()` was being invoked implicitly. I could have left as it was, but wanted to have the ability of seeing all callsites converting `StructuredMessage` to string.